lib/tokenize/block-quote.js

```diff
@@ -1,5 +1,8 @@
-exports.tokenize = tokenizeBlockQuoteStart
-exports.continuation = {tokenize: tokenizeBlockQuoteContinuation}
+var tokenize = tokenizeBlockQuoteStart
+var continuation = {tokenize: tokenizeBlockQuoteContinuation}
```

Comment:
Would it work to have a `var blockQuote = {tokenize: tokenize, continuation: continuation, exit: exit}` here, export that, and use it in the continuation?

Or only this at the top:

```
var blockQuote = {
  tokenize: tokenizeBlockQuoteStart,
  continuation: tokenizeBlockQuoteContinuation,
  exit: exit
}
```

...and exporting that, and using that in continuation?

Reply:
So you suggest to make every tokenize a default export?